multipass-addressing/README.md

```diff
@@ -0,0 +1,52 @@
+![Precisely](../Precisely_Logo.png)
+
+# Multipass Addressing Sample
+This sample for the Spark Verify and Geocoding API in Scala demonstrates how to improve geocoding results by performing verify first and then geocoding. With this multipass-addressing, for all first-pass results without point-level precision, a second geocoding pass is run using single line address matching, which may return more accurate geocodes.
```

Comment:
This sample for the Spark Addressing SDK in Scala... 
With this multipass addressing example, ...
..., which may increase match rate.
(be careful with "accurate" here)

Reply:
Ok